src/Source/AbstractSource.php

```diff
@@ -100,12 +100,12 @@ protected function getMappedItemData(?array $element, array $mapping): array
         return $result;
     }
 
-    protected function getContentFromUrl(string $method, string $url, array $auth = [], array $HTTPClientOptions = []): array
+    protected function getContentFromUrl(string $method, string $url, array $options = []): array
```

Comment:
Please rename $options to $httpClientOptions since we might have a general $options array sometimes. (same below)